WorldTrotter/ConversionViewController.swift

```diff
@@ -26,17 +27,31 @@ class ConversionViewController: UIViewController{ //, UITextFieldDelegate {
         let existingTextHasDecimalSeparator = textField.text?.range(of: ".")
         let replacementTextHasDecimalSeparator = string.range(of: ".")
         
-        if existingTextHasDecimalSeparator != nil && replacementTextHasDecimalSeparator != nil {
+        // assign a variable with a CharacterSet 'letters'
+        let letters = CharacterSet.letters
+        // assign a variable to
+        let stringHasLetters = string.rangeOfCharacter(from: letters)
+    
+        
+        if existingTextHasDecimalSeparator != nil, replacementTextHasDecimalSeparator != nil {
```

Comment:
Though this works, typically the comma notation is used when you are performing conditional assignment, as in optional binding.

Reply:
Refactor the two if statements into a single compound condition using (AND) && and (OR) ||